static/404.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <title>
+        404 Error | Sauce Docs
+    </title>
+    <link rel="stylesheet" href="/css/404.css">
+    <link data-react-helmet="true" rel="shortcut icon" href="/img/favicon.ico">
+</head>
+<body>
+    <div class="container">
+        <div class="flex header"><h2>404 Error | Sauce Docs: Content Does Not Exist!</h2></div>
```

Comment:
Do we want to say "content does not exist"? That seems a little bleak. Maybe something like "We can't find that page" or even "that URL does not exist"? 

Sorry to be such a word-nerd. I just would like to believe that _the content_ probably does exist somewhere, just not at the URL you've entered. :)

Reply:
```suggestion
        <div class="flex header"><h2>404 Error | Sauce Docs: URL does not exist</h2></div>
```